modules/engage-paella-player/package.json

```diff
@@ -2,20 +2,49 @@
   "name": "paella-opencast",
   "private": true,
   "scripts": {
-    "build": "gulp paella-opencast:build",
+    "build": "gulp paella-opencast:build.dce",
     "eslint": "eslint --config ../../docs/checkstyle/eslintrc.js src/main/paella-opencast/ gulpfile.js --resolve-plugins-relative-to .",
     "html-linter": "html-linter --config ../../docs/checkstyle/html-linter.json 'src/main/resources/**/*.html'"
   },
-  "devDependencies": {
-    "eslint": "^8.27.0",
+  "dependencies": {
+    "cors": "^2.8.5",
+    "dce-paella-extensions": "harvard-dce/dce-paella-extensions#t/OPC-682-HUDCE-Paella7-upgrade-collab",
```

Comment:
This is temporary while doing the Paella 7 upgrade. Branch `t/OPC-682-HUDCE-Paella7-upgrade-collab` matches current main/master DCE paella extensions